src/main/kotlin/pt/up/fe/ni/website/backend/service/AccountService.kt

```diff
@@ -38,4 +61,13 @@ class AccountService(private val repository: AccountRepository, private val enco
         account.password = encoder.encode(dto.newPassword)
         repository.save(account)
     }
+
+    fun deleteAccountById(id: Long): Map<String, String> {
+        if (!repository.existsById(id)) {
+            throw NoSuchElementException(ErrorMessages.accountNotFound(id))
+        }
+
+        repository.deleteById(id)
+        return emptyMap()
+    }
```

Comment:
We shouldn't return emptyMap() in the service (we changed this recently). Services should not handle API responses, that's for controllers